sites/partners/src/listings/PaperListingDetails/sections/helpers.ts

```diff
@@ -16,3 +16,23 @@ export const getDetailFieldDate = (listingDate: Date) => {
 export const getDetailFieldTime = (listingTime: Date) => {
   return listingTime ? moment(new Date(listingTime)).format("hh:mm:ss A") : t("t.none")
 }
+
+export const getReadableErrorMessage = (errorMessage: string | undefined) => {
```

Comment:
can you also create unit tests, or an issue to create them for this function too.

Reply:
We don't have unit tests in partners rn but I will create an issue